05LessonFive/Checkpoint1/index.html

```diff
@@ -5,6 +5,22 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+<main>
+  <section id="picture">
```

Comment:
for the picture and the text, you've double-nested sections and divs, so right now it's section--> div --> content. could you take either the section or div out?